sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 3.2.0-beta.1 (Unreleased)
-
+### Breaking Changes
+- Change the default format of the AAD B2C redirect url to `{BaseUrl}/login/oauth2/code/`.
```

Comment:
IMU, `Breaking changes` only appears in major version upgrade.
Refs: https://semver.org/#spec-item-8